.github/labeler.yml

```diff
@@ -0,0 +1,26 @@
+# Label to be added when the PR is updated:
+"Awaiting Review":
+  - "**/*"
+
+# Labels to be removed when the PR is updated:
+"Reviewed: Ready to Merge": []
+"Reviewed: Action Needed": []
+
+# Labels based on which files are modified:
+"Gem Updates":
+  - "Gemfile"
+  - "Gemfile.lock"
+
+"Has Migrations":
+  - "db/migrate/**/*"
+
+"Has Production Config Changes":
+  - "config/config.yml"
+
+"Scope: i18n Only":
+  - all:
+    - "config/locales/**/*"
+
+"Scope: Tests Only":
+  - all:
+    - "+(factories|features|spec|test)/**/*"
```

Comment:
Can you also add `.github` here? No worries about the CI files under config/ and script/ though.